Source/WTF/wtf/Vector.h

```diff
@@ -662,7 +663,7 @@ struct UnsafeVectorOverflow {
 // Template default values are in Forward.h.
 template<typename T, size_t inlineCapacity, typename OverflowHandler, size_t minCapacity, typename Malloc>
 class Vector : private VectorBuffer<T, inlineCapacity, Malloc> {
```

Comment:
Check JSC `runtime/CachedTypes.cpp`'s VectorMalloc too. Probably it should be VectorBufferMalloc.